README.rst

```diff
@@ -51,7 +51,10 @@ It basically means:
 - generate metadata for haskell repo after main
   tree sync is done, using N+1 cores
 
-Build issues? If you see packages from the gentoo repository being
```

Comment:
This looks good to me, although I think something along the lines of "Overlay Priority" would be a better heading than "Build issues?".

Reply:
I agree! I have rewritten the section considering your comments, and rebased, so hopefully now OK!